### PR TITLE
6 refactor test mvp.ipynb

### PR DESCRIPTION
**Pull request by eriktks**
Ref: #6 

* Refactored notebook test_mvp.ipynb: split main code block in smaller parts and added more comments
* Additionally added environment include line for Jupyter Lab to README

To test: install notebook according to installation instructions in README.md and verify that the notebook works

### CODE REVIEW

File: test_mvp.ipynb [hunk 1]
```diff
@@ -2,56 +2,157 @@
  "cells": [
```
Comment (sverhoeven) [inline test_mvp.ipynb]: <p>Can we add a <code># header</code> here so its clear we are switching subjects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sequgen/notebooks/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (sverhoeven) [inline test_mvp.ipynb]: <p>This plot looks very similar to the previous, I expected a the peak at 4.6 not after 5. Are you sure the plot is up to date with the sampled parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sequgen/notebooks/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='21'/>

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Added some comments to https://app.reviewnb.com/sequgen/notebooks/pull/8/

File: test_mvp.ipynb (hunk 1 above)
Comment (eriktks) [reply test_mvp.ipynb]: <p>We found out that placement refers to the start of the peak (left part of triangle)</p>

Comment (sverhoeven) [reply test_mvp.ipynb]: thanks for the explaination

**Review by sverhoeven — APPROVED** [review]
Looks good

**Comment by eriktks** [comment]
Thanks!

**Pull request merged**